specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -2452,6 +2455,11 @@
         "allowSharedKeyAccess": {
           "type": "boolean",
           "description": "Indicates whether the storage account permits requests to be authorized with the account access key via Shared Key. If false, then all requests, including shared access signatures, must be authorized with Azure Active Directory (Azure AD). The default value is null, which is equivalent to true."
+        },
+        "isNfsV3Enabled": {
+          "type": "boolean",
+          "x-ms-client-name": "EnableNfsV3",
+          "description": "NFS 3.0 protocol support enabled if sets to true."
```

Comment:
we could disable it with false value, right?

Reply:
No. This is immutable. Similar to isHnsEnabled.